### PR TITLE
Check SUSEConnect --status-text and save output info to logfile

### PR DESCRIPTION
**Pull request by tinawang123**
Check SUSEConnect --status-text and add info to log

- Related ticket: https://progress.opensuse.org/issues/65211
- Verification run: 
  https://openqa.nue.suse.com/tests/4096510#
  https://openqa.nue.suse.com/tests/4096511#

### CODE REVIEW

File: tests/console/system_prepare.pm [hunk 1]
```diff
@@ -70,6 +70,9 @@ sub run {
         change_grub_config('=.*', '=1024x768x32', 'GFXPAYLOAD_LINUX=');
         grub_mkconfig;
     }
+
```
Comment (lemon-suse) [inline tests/console/system_prepare.pm:73]: Please add comments here for the reason why need the code.

File: tests/migration/sle12_online_migration/post_migration.pm [hunk 2]
```diff
@@ -25,6 +25,9 @@ sub run {
     # print repos to screen and serial console after online migration
     zypper_call('lr -u');
 
+    my $out = script_output("SUSEConnect --status-text", proceed_on_failure => 1);
```
Comment (lemon-suse) [inline tests/migration/sle12_online_migration/post_migration.pm:28]: Same here.

**Comment by lemon-suse** [comment]
Please update the PR description and title, the reason why need the PR, and it should be 'save output info to logfile'. Please update the git commit also.

File: tests/console/system_prepare.pm [hunk 3]
```diff
@@ -70,6 +70,10 @@ sub run {
...
+    # check SUSEConnect --status-text and save output info to logfile
+    my $out = script_output("SUSEConnect --status-text", proceed_on_failure => 1);
```
Comment (lemon-suse) [inline tests/console/system_prepare.pm:75]: No check for the output of 'SUSEConnect --status-text'.

**Comment by tinawang123** [comment]
Verified 
online: https://openqa.nue.suse.com/tests/4106356
offline: https://openqa.nue.suse.com/tests/4106355
media: https://openqa.nue.suse.com/tests/4106357

**Comment by lemon-suse** [comment]
Please resolve the CI issue firstly.

**Comment by DimStar77** [comment]
Please revert or fix the fallout on Tumbleweed, e.g. https://openqa.opensuse.org/tests/1233461#step/system_prepare/13

**Comment by nilxam** [comment]
also breaks Leap release process https://openqa.opensuse.org/tests/1233637#step/system_prepare/8

**Pull request merged**